core/src/main/java/org/polypheny/db/sql/SqlKind.java

```diff
@@ -1216,7 +1219,37 @@
      *
      * <b>Note to other projects</b>: If you are extending Polypheny-DB's SQL parser and have your own object types you no doubt want to define CREATE and DROP commands for them. Use OTHER_DDL in the short term, but we are happy to add new enum values for your object types. Just ask!
      */
-    OTHER_DDL;
+    OTHER_DDL,
+
+    DOC_VALUE,
+
+    DOC_ITEM,
+
+    DOC_SIZE_MATCH,
+
+    DOC_REGEX_MATCH,
+
+    DOC_TYPE_MATCH,
+
+    DOC_SLICE,
+
+    DOC_EXCLUDE,
+
+    DOC_ELEM_MATCH,
+
+    DOC_UNWIND,
+
+    DOC_UPDATE_REPLACE,
+
+    DOC_UPDATE_ADD,
+
+    DOC_UPDATE_REMOVE,
+
+    DOC_UPDATE_RENAME,
+
+    DOC_UPDATE,
+
+    DOC_EXISTS;
```

Comment:
Why do we have document stuff here?

Reply:
Due to the fact that SqlKind was still used to determen the type of the RelNodes, the additional operators needed to be added here (The name of the class is a bit missleading in my opinion). With the upcoming refactor this class will be renamed into a more generic Kind or PolyKind class.